src/key.rs

```diff
@@ -66,7 +66,7 @@ impl RedisKey {
         self.key_inner == null_key
     }
 
-    pub fn read(&self) -> Result<Option<String>, Error> {
+    pub fn read(&self) -> Result<Option<String>, RedisError> {
```

Comment:
We need to tidy up our usage of `Error` vs `RedisError`. Need to think about this for a bit.